README.md

```diff
@@ -132,7 +132,7 @@ setup a first admin. To do this:
 
 1.  Create a user account by registering through the web interface
 2.  Attach to the docker image for the postgres database into pg:
-    `docker container exec -it lilflosystem_postgres_1 psql flodb -h localhost -U postgres`
+    `docker container exec -it flosystem_postgres_1 psql flodb -h localhost -U postgres`
```

Comment:
When I had to do this, the name of the container running the database was flosystem-postgres-1 , can you check on this please?

Reply:
edited